fakeData.js

```diff
@@ -0,0 +1,93 @@
+const psychiarists = [
+    {
+        doctorImg: "https://images.all-free-download.com/images/graphiclarge/doctor_work_painting_colored_cartoon_character_sketch_6847622.jpg",
+        name: "Dr. Nabeel No'man",
+        specialtyies: "Anxiety, Bipolar Disorder, Depression, General Adult Psychiatry, Personality Disorders",
```

Comment:
From the design looks like we will only display one thing. So Let's reflect that her and just pick one thing to add.